recipes-core/lcd-tools/lcd-tools_0.1.bb

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION = "Small wrapper to sync LCD time"
+PR = "r0"
+DEPENDS = "libhybris"
+SRC_URI = "file://lcd-sync-time.service \
+           file://lcd-sync-time.timer \
+           file://lcd-sync-time.cpp \
+           file://lcd-tools.cpp \
+           file://lcd-tools.h \
+           file://CMakeLists.txt \
+           file://COPYING \
+           git://github.com/jarro2783/cxxopts;protocol=https;tag=v3.0.0;destsuffix=third-party/cxxopts/"
```

Comment:
Please make a separate recipe for cxxopts or use another library already provided by an OpenEmbedded recipe. (for AsteroidOS, qtbase is a good choice)
This is technically a different project and under a different license (MIT).

Reply:
Where do you suggest to put the recipe.
For this one I would prefer a more lightweight non-qt interface. Mostly because it would not require as much rebuilding when the base changes.

Reply:
So I've found `cli11` which can serve as the alternative for this. It is not the `2.x` branch, but it's better than nothing. I will also plan on using `sdbuc-c++`. Mostly I want to avoid qt dependence when possible so that the source is easy to understand for new-comers. Do you agree with this approach?